tests/core/go_proto_library/BUILD.bazel

```diff
@@ -117,9 +117,16 @@ go_test(
     deps = [":protos_go_proto"],
 )
 
+go_test(
```

Comment:
Mention this in README.rst with a brief explanation.

Reply:
done!